src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -0,0 +1,66 @@
+#ifndef HALIDE_HALIDERUNTIMEHEXAGONHOST_H
+#define HALIDE_HALIDERUNTIMEHEXAGONHOST_H
+
+#include "HalideRuntime.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** \file
+ *  Routines specific to the Halide Hexagon host-side runtime.
+ */
+
+typedef int halide_hexagon_handle_t;
+
+extern const struct halide_device_interface *halide_hexagon_device_interface();
+
+/** Set the underlying ion device handle for a buffer. The device
```

Comment:
This comment is a bit stale. I originally had these buffers storing ION allocator file handles (https://android.googlesource.com/platform/system/core/+/master/libion/include/ion/ion.h). However, they now just store pointers, so the detail is not necessary.